test/buffer/worker/expected-etcd-state/000-setup.txt

```diff
@@ -0,0 +1,6 @@
+<include _runtime>
+<include _config>
+<include _secrets>
+<include _setup_tasks>
+<include _opened_files>
+<include _opened_slices>
```

Comment:
Test bude fungovat tak, ze sa budu robit nejake operacie a skontroluje sa vzdy stav `etcd`.

Aby to nebolo neprehladne, kedze vela casti sa bude opakovat, tak som tu pridal moznosti `include`.

Toto je prvy stav, ktory tu je, stav po vytvoreni `receiver` + 2x `export`.